index.html

```diff
@@ -9323,8 +9325,8 @@ <h3 id="example-2">
             </li>
           </ul>
           <p>
-            Now, the actual algorithm to determine the reduction gain, in
-            linear gain value.
+            Now, the actual algorithm to determine <var>reduction gain</var>, in
+            linear gain value, for each sample of input.
```

Comment:
This isn't a complete sentence.  Maybe just replace the final period with a colon?